docs/antrea-ipam.md

```diff
@@ -214,3 +214,165 @@ IP, because inter-Node traffic of AntreaIPAM Pods is forwarded by the Node netwo
 router should provide the network connectivity for these VLANs. Only a single IP pool can
 be included in the Namespace annotation. In the future, annotation of up to two pools for
 IPv4 and IPv6 respectively will be supported.
+
+## IPAM for Secondary Network
+
+With the AntreaIPAM feature, Antrea can allocate IPs for Pod secondary networks. At the
+moment, AntreaIPAM supports secondary networks managed by [Multus](https://github.com/k8snetworkplumbingwg/multus-cni),
+we will add support for [secondary networks managed by Antrea](feature-gates.md#secondarynetwork)
+in future.
+
+### Prerequisites
+
+The IPAM capability for secondary network was added in Antrea version 1.7. It
+requires the `AntreaIPAM` feature gate to be enabled on both `antrea-controller`
+and `antrea-agent`, as `AntreaIPAM` is still an alpha feature at this moment and
+is not enabled by default.
+
+### CNI IPAM configuration
+
+To configure Antrea IPAM, `antrea` should be specified as the IPAM plugin in the
+the CNI IPAM configuration, and at least one Antrea IPPool should be specified
+in the `ippools` field. IPs will be allcoated from the specified IPPool(s) for
+the secondary network.
+
+```json
+{
+    "cniVersion": "0.3.0",
+    "name": "ipv4-net-1",
+    "type": "macvlan",
+    "master": "eth0",
+    "mode": "bridge",
+    "ipam": {
+        "type": "antrea",
+        "ippools": [ "ipv4-pool-1" ]
+    }
+}
+```
+
+Multiple IPPools can be specified to allocate multiple IPs from each IPPool for
+the secondary network. For example, you can specify one IPPool to allocate an
+IPv4 address and another IPPool to allocate an IPv6 address in the dual-stack
+case.
+
+```json
+{
+    "cniVersion": "0.3.0",
+    "name": "dual-stack-net-1",
+    "type": "macvlan",
+    "master": "eth0",
+    "mode": "bridge",
+    "ipam": {
+        "type": "antrea",
+        "ippools": [ "ipv4-pool-1", "ipv6-pool-1" ]
+    }
+}
+```
+
+Additionally, Antrea IPAM also supports configuration of static IP addresses,
+static routes, and DNS settings, as what are supported by the [static IPAM
+plugin](https://www.cni.dev/plugins/current/ipam/static). The following example
```

Comment:
That sentence is not correct. Maybe you mean:

> Additionally, Antrea IPAM also supports the same configuration of static IP addresses, static routes, and DNS settings as what is supported by the ...

Reply:
Updated.